submission/persistence-filesystem/src/main/kotlin/ac/uk/ebi/biostd/persistence/filesystem/service/StorageService.kt

```diff
@@ -32,19 +32,17 @@ class StorageService(
         }
 
     override suspend fun releaseSubmissionFile(
+        sub: ExtSubmissionInfo,
         file: ExtFile,
-        subRelPath: String,
```

Comment:
@jhoanmanuelms this api was changed to be consistent with others, as they all recieve a submission but this one did not follow the same pattern but rather explicit realpath and secret key which make the code incosistent